azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/toolkit/lib/common/database/FirewallRuleEntity.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.lib.common.database;
+
+import com.microsoft.azure.toolkit.lib.common.entity.IAzureResourceEntity;
+import com.microsoft.azure.toolkit.lib.common.utils.NetUtils;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.apache.commons.lang3.StringUtils;
+
+@Getter
+@SuperBuilder(toBuilder = true)
+public class FirewallRuleEntity implements IAzureResourceEntity {
+
+    private static final int MAX_FIREWALL_NAME_LENGTH = 128;
+    public static final String ACCESS_FROM_LOCAL_FIREWALL_RULE_NAME =  FirewallRuleEntity.getAccessFromLocalFirewallRuleName();
+    public static final String ACCESS_FROM_AZURE_SERVICES_FIREWALL_RULE_NAME = "AllowAllWindowsAzureIps";
+    public static final String IP_ALLOW_ACCESS_TO_AZURE_SERVICES = "0.0.0.0";
+
+    private final String name;
+    private final String id;
+    private final String subscriptionId;
+    private final String startIpAddress;
+    private final String endIpAddress;
+
+    private static String getAccessFromLocalFirewallRuleName() {
```

Comment:
avoid to use static field to save this method call

Reply:
updated